exporter.go

```diff
@@ -207,10 +265,13 @@ func main() {
 		prometheus.GaugeOpts{
 			Name: "nginxexporter_build_info",
 			Help: "Exporter build information",
-			ConstLabels: prometheus.Labels{
-				"version":   version,
-				"gitCommit": gitCommit,
-			},
+			ConstLabels: collector.MergeLabels(
+				prometheus.Labels{
```

Comment:
in `collector/nginx_plus.go` you mrge labels in a different order. For example, `MergeLabels(constLabels, prometheus.Labels{"code": "4xx"})`
I suggest you change the order here , so that the labels that are provided via a command-line argument / ENV variable cannot override any built-in labels.